HISTORY.rst

```diff
@@ -2,6 +2,30 @@
 History
 =======
 
+3.6.0 (2020-02-06)
+------------------
+Changes:
+
+* `cci task info` now has Command Syntax section and improved formatting of option information.
```

Comment:
@Br4nd0R can you run `make docs`, confirm the new generated docs get formatted successfully as html, and check in the resulting docs/tasks.rst?

Reply:
Yup, Task Reference section looks great!

Reply:
@Br4nd0R so are you going to check it in so that it will render on readthedocs?